x/slashrefund/keeper/deposit_test.go

```diff
@@ -52,9 +24,10 @@ func TestDepositGet(t *testing.T) {
 		require.Equal(t, deposit, rst)
 	}
 }
+
 func TestDepositRemove(t *testing.T) {
```

Comment:
Should be renamed in TestRemoveDeposit since the tested function is RemoveDeposit().

Reply:
right